packages/protocol-contracts/contracts/ZetaConnector.eth.sol

```diff
@@ -3,23 +3,24 @@ pragma solidity 0.8.7;
 
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
+import "./interfaces/ConnectorErrors.sol";
 import "./ZetaConnector.base.sol";
-import "./ZetaInterfaces.sol";
+import "./interfaces/ZetaInterfaces.sol";
 
 contract ZetaConnectorEth is ZetaConnectorBase {
     constructor(
-        address zetaTokenAddress_,
-        address tssAddress_,
-        address tssAddressUpdater_
-    ) ZetaConnectorBase(zetaTokenAddress_, tssAddress_, tssAddressUpdater_) {}
+        address zetaTokenAddress,
```

Comment:
if we change the param I mention before we should change this to be consistent. the last `_` was because the static analysis recommend to use the same name as inherited contract

Reply:
Done